APITesters/ObjCAPITester/ObjCAPITesterIOS/SceneDelegate.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  SceneDelegate.swift
+//  ObjCAPITesterIOS
+//
+//  Created by Andrés Boedo on 10/6/21.
+//
+
+import UIKit
+
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+
+    var window: UIWindow?
```

Comment:
would it be possible for the objctester and the swiftester to reuse the same host app?

Reply:
ah ok, was just curious. thanks for the explanation